Index: app/src/main/java/com/example/pic_pho/CellularSocket/SocketActionIntentService.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/example/pic_pho/CellularSocket/SocketActionIntentService.kt	(date 1617088406000)
+++ app/src/main/java/com/example/pic_pho/CellularSocket/SocketActionIntentService.kt	(date 1617088406000)
@@ -0,0 +1,22 @@
+package com.example.pic_pho.CellularSocket
+
+import android.app.IntentService
+import android.content.Intent
+import android.content.Context
+import android.util.Log
+import java.lang.Thread.sleep
+
+
+class SocketActionIntentService : IntentService("SocketActionIntentService") {
+
+    private val TAG = "SocketActionIntentServi"
+
+
+    override fun onHandleIntent(intent: Intent?) {
+
+
+
+    }
+
+
+}
\ No newline at end of file
